Makefile

```diff
@@ -209,6 +260,25 @@ pack:
 
 #----------------------------------------------------------------------------------------------
 
+COV_EXCLUDE_DIRS += deps tests
```

Comment:
Can these move into readies/mk/coverage* and maybe add a flag for USES_RUST or similar?